.github/styles/Blog/Typos.yml

```diff
@@ -0,0 +1,11 @@
+extends: substitution
+message: Do you mean '%s' instead of '%s'?
+level: warning
+ignorecase: true
+# swap maps tokens in form of bad: good
+# for more information: https://vale.sh/docs/topics/styles/#substitution
+swap:
+  bellow: below
+  eg: e.g.
+  eg.: e.g.
+  'e.g ': 'e.g. '
```

Comment:
this reminds me: can we add a rule to avoid Latinisms altogether? (etc., e.g., i.e.) - they're often confusing to non-native English speakers, and the latter two in particular are frequently misused. "etc." can always be cut or replaced with "and more", and "e.g" and "i.e." can be replaced with "for example."

Reply:
Yes, Google has a Latin packge to detect `eg` and `ie`: https://github.com/errata-ai/Google/tree/master/Google